docs/AppService.md

```diff
@@ -67,3 +67,175 @@ Run the Validation Test
+- contentLength - Content length of test response
+- errorCount - Number of errors encountered running the test
+
+Refer to the Log Analytics Kusto Query Language (KQL) [overview](https://docs.microsoft.com/en-us/azure/data-explorer/kusto/query/) for further information on creating queries.
+
+```bash
+
+# Get 10 most recent log entries
+ContainerInstanceLog_CL | top 10 by TimeGenerated
+
+# Unpack all fields of Message object into columns
+ContainerInstanceLog_CL
+| extend jsonMessage = parsejson(Message)
+| extend category = tostring(jsonMessage.category),
+  path = tostring(jsonMessage.path),
+  tag = tostring(jsonMessage.tag),
+  code = toint(jsonMessage.statusCode),
+  duration = toint(jsonMessage.duration),
+  quartile = toint(jsonMessage.quartile),
+  logType = tostring(jsonMessage.logType),
+  length = toint(jsonMessage.contentLength),
+  errors = toint(jsonMessage.errorCount)
+
+# Failed or error tests
+ContainerInstanceLog_CL
+| extend jsonMessage = parsejson(Message)
+| extend errors = toint(jsonMessage.errorCount),
+  code = toint(jsonMessage.statusCode)
+| where code != 200 or errors > 0
+
+# Test runs exceeding 2000ms
+ContainerInstanceLog_CL
+| extend jsonMessage = parsejson(Message)
+| extend duration = toint(jsonMessage.duration)
+| where duration > 2000
+
+# Average duration of tests per category and location
+ContainerInstanceLog_CL
+| extend jsonMessage = parsejson(Message)
+| extend category = tostring(jsonMessage.category),
+  duration = toint(jsonMessage.duration),
+  tag = tostring(jsonMessage.tag)
+| where isnotempty(category)
+| summarize avg(duration) by category, tag
+| sort by category
+
+# Average quartile of tests per category and location
+ContainerInstanceLog_CL
+| extend jsonMessage = parsejson(Message)
+| extend category = tostring(jsonMessage.category),
+  quartile = toint(jsonMessage.quartile),
+  tag = tostring(jsonMessage.tag)
+| where isnotempty(category)
+| summarize avg(quartile) by category, tag
+| sort by category
+
+# Average quartile of tests per category and location
+ContainerInstanceLog_CL
+| extend jsonMessage = parsejson(Message)
+| extend category = tostring(jsonMessage.category),
+  duration = toint(jsonMessage.duration),
+  tag = tostring(jsonMessage.tag)
+| where isnotempty(category)
+| summarize percentile(duration, 98) by category, tag
+| sort by category
+
+# Tests with duration exceeding the 98th percentile within the matching category
+ContainerInstanceLog_CL
+| extend jsonMessage = parsejson(Message)
+| extend category = tostring(jsonMessage.category),
+  duration = toint(jsonMessage.duration)
+| join kind=leftouter (
+    ContainerInstanceLog_CL
+    | extend jsonMessage = parsejson(Message)
+    | extend category = tostring(jsonMessage.category),
+      duration = toint(jsonMessage.duration)
+    | where isnotempty(category)
+    | summarize percentile(duration, 98) by category
+  )
+  on category
+| where duration > percentile_duration_98
+| project category, duration, jsonMessage
```

Comment:
Was hoping for some nested query where I can just compare the duration to the percentile_duration. I'll keep looking for another way, rather than doing this join.